Tests/Core/AWSClientRuntimeTests/Auth/CredentialsProvidersTests/SSOCredentialsProviderTests.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import ClientRuntime
+import Foundation
+import XCTest
+
+@_spi(FileBasedConfig) @testable import AWSClientRuntime
+
+class SSOCredentialsProviderTests: XCTestCase {
+    let configPath = Bundle.module.path(forResource: "sso_tests", ofType: nil)!
+    let credentialsPath = Bundle.module.path(forResource: "credentials", ofType: nil)!
+    
+    func testCreateCredentialsProviderSSOLegacyProfile() async throws {
+        _ = try SSOCredentialsProvider(
+            profileName: "user",
+            configFilePath: configPath,
+            credentialsFilePath: credentialsPath
+        )
+        // SUCCESS: creation didn't throw error
```

Comment:
Nothing we can do to verify that the expected profile & config files are accessed?

Reply:
Found a workaround way to verify the expected profile and config files are accessed by adding a test case that throws exception due to having a nonexistent profile name as one of the inputs to the SSO credentials provider constructor.